### DISCUSSION TITLE
为什么运行web_demo.py后出现一个网址，点进去显示无法访问此网页啊？

### DISCUSSION THREAD

**Discussion by yitiaolittle** [zai-org/ChatGLM3#562] [Q&A] [answered]
![微信图片_20231208152432](https://github.com/THUDM/ChatGLM3/assets/105427014/1374180c-db0c-461d-a1e0-9c2b6a29d4ba)
为什么网址点进去会显示这个啊？出不来对应的界面

**Comment by zRzRzRzRzRzRzR** [comment]
看一下streamlit 填邮箱

**Reply by yitiaolittle** [reply]
![image](https://github.com/THUDM/ChatGLM3/assets/105427014/f95991fb-3f6c-4de2-8221-d3457c3f55ed)

邮箱：zwzybzj11@163.com

**Reply by zRzRzRzRzRzRzR** [reply]
不是，你运行的时候会让你填邮箱，命令行

**Reply by yitiaolittle** [reply]
？？？我运行的时候没有让我填邮箱啊，我在终端输入 python web_demo.py 然后会自动给我出现一个网址，我点进去之后显示无法访问此页面。请问是我命令输错了吗？？
![cc742026a635b2a37a4753614130a9f](https://github.com/THUDM/ChatGLM3/assets/105427014/bcd83886-9638-4cf2-af7a-bdf1f42b85a9)

**Reply by zRzRzRzRzRzRzR** [reply, marked as answer]
你这不是gradio吗...  gradio要降级到3.x
